scripts/update_chromium_revision.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2025 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+
+import os
+import requests
+from datetime import date
+from google.cloud import storage
+from google.cloud import secretmanager
+
+
+BUCKET_NAME = 'wpt-versions'
+NEW_REVISION_FILE = 'pinned_chromium_revision_NEW'
+OLD_REVISION_FILE = 'pinned_chromium_revision'
+REPO_OWNER = "web-platform-tests"
+REPO_NAME = "wpt"
+PR_NUMBER = "50375"
+
+
+def all_passing_checks() -> bool:
+    """Check if all CI tests passed."""
+    s = requests.Session()
+    s.headers.update({
+        "Authorization": f"token {get_token()}"
+    })
+    url = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/commits/{get_sha()}/check-suites"
```

Comment:
We should either:
1. Use the GitHub client python library
2. Manually [pin](https://docs.github.com/en/rest/about-the-rest-api/api-versions?apiVersion=2022-11-28) the `X-GitHub-Api-Version` header in each API call.

Right now, we are expecting the response to be a certain format. If the API version changes and the shape of the response changes, we will start to get KeyError. (for example when we go to check the conclusion.

Reply:
Good idea. I think I'll approach using #2 and make changes once I confirm everything functions as expected.